pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -49,9 +58,15 @@ func (pl *ImageLocality) Score(ctx context.Context, state *framework.CycleState,
 		return 0, framework.NewStatus(framework.Error, fmt.Sprintf("getting node %q from Snapshot: %v", nodeName, err))
 	}
 
-	meta := migration.PriorityMetadata(state)
-	s, err := priorities.ImageLocalityPriorityMap(pod, meta, nodeInfo)
-	return s.Score, migration.ErrorToFrameworkStatus(err)
+	nodeInfos, err := pl.handle.SnapshotSharedLister().NodeInfos().List()
```

Comment:
I think we still need a post-filter plugin for calculating totalNumNodes

cc/ @ahg-g

Reply:
You don't need a PreFilter for that, it is just ```len(sharedLister.NodeInfos().List())```